monkey/common/agent_plugins/agent_plugin_manifest.py

```diff
@@ -14,6 +14,7 @@ class AgentPluginManifest(InfectionMonkeyBaseModel):
         :param name: Plugin name in snake case
         :param plugin_type: Type of the plugin (exploiter, fingerprinter,
          credential collector, etc.)
+        :param supported_operating_systems: List of operating systems the plugin can run on
```

Comment:
Might be confusing, because it's not a "list", it's a tuple. Might be better to just say "Operating systems the plugin can run on"